src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -746,6 +746,15 @@ void java_bytecode_languaget::replace_string_methods(
       // Replace body of the function by code generated by string preprocess
       symbolt &symbol=context.lookup(id);
       symbol.value=generated_code;
+      // Specifically instrument the new code, since this comes after the
+      // blanket instrumentation pass called before typechecking.
+      java_bytecode_instrument(
```

Comment:
Since there was already something relying on this instrumentation pass [java_bytecode_convert_method.cpp:527](https://github.com/diffblue/cbmc/pull/1229/files#diff-489726e05da896c8c706122383a4744eL527) I was expecting to see a removal to this call as it is moved to later in the process, but it seems we are just adding one?